admin/bootstrap.py

```diff
@@ -84,10 +86,12 @@ def clean_up_old_tails_venv(virtualenv_dir: str = VENV_DIR) -> None:
     venv, so it'll get recreated.
     """
     if is_tails():
-        try:
-            dist = subprocess.check_output("lsb_release --codename --short", shell=True).strip()
-        except subprocess.CalledProcessError:
-            return None
+        with open("/etc/os-release") as f:
+            os_release = f.readlines()
+            for line in os_release:
+                if line.startswith("VERSION_CODENAME="):
+                    dist = line.split("=")[1].strip().strip('"')
+                    break
 
         # Tails 5 is based on bullseye / Python 3.9
         if dist == b"bullseye":
```

Comment:
Here we're trying to detect whether we're on Tails 5. So we can look for `TAILS_VERSION_ID` and see if it `.startswith('5.')` probably.

Reply:
I did the same but the `python3_buster_venv_deleted_in_bullseye` test is failing. I am not sure why the script is unable to delete the Tails 4 virtualenv.

Reply:
I think we just need to adjust the mocking, let me see...